meta/and.rb

```diff
@@ -1,4 +1,4 @@
-Mutant::Meta::Example.add do
+Mutant::Meta::Example.add :and do
```

Comment:
I'm surprised that you opted to write these all without parenthesis as opposed to `Mutant::Meta::Example.add(:and) do`